source/features/convert-release-to-draft.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import GitHubURL from '../github-helpers/github-url';
+import LoadingIcon from '../github-helpers/icon-loading';
+import {getRepoURL, getRepoGQL} from '../github-helpers';
+
+const getReleaseID = async (): Promise<string> => {
+	const tagName = location.pathname.split('/').pop()!;
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			release(tagName: "${tagName}") {
+				id
+			}
+		}
+	`);
+
+	// => atob(repository.release.id) => "07:Release27300592"
+	return atob(repository.release.id).replace(/.*Release/, '');
```

Comment:
This is obscene (not your fault)

What do you think if we just use api.v3 for this too? You could even just inline it in `convertToDraft`

Reply:
Its a **much** longer request. I tried that first.

Reply:
Its a **much** longer request. I tried that first.